doc/source/user/quickstart.rst

```diff
@@ -628,10 +628,18 @@ An array has a shape given by the number of elements along each axis::
 
 The shape of an array can be changed with various commands::
 
-    >>> a.ravel() # flatten the array
+    >>> a.ravel() # returns the array, flattened
     array([ 2.,  8.,  0.,  6.,  4.,  5.,  1.,  1.,  8.,  9.,  3.,  6.])
-    >>> a.shape = (6, 2)
-    >>> a.T
+    >>> a.ravel().shape
+    (12,)
+    >>> a.reshape(6,2) # returns the array with a modified shape
+    array([[ 2.,  8.],
+           [ 0.,  6.],
+           [ 4.,  5.],
+           [ 1.,  1.],
+           [ 8.,  9.],
+           [ 3.,  6.]])
+    >>> a.T # returns the array after being transposed
```

Comment:
Hi, I understand what you're talking about and hence my comment that it returns the array with a new shape rather than manipulate it.. 
I think that's the case for `a.ravel() and a.T` as well. All of the three examples doesn't change `a`